spec/models/generic_file_spec.rb

```diff
@@ -8,6 +8,12 @@
     @file.apply_depositor_metadata(user.user_key)
   end
 
+  it 'is initialized with blanks' do
```

Comment:
It's to illustrate the changes made to the `FitsDatastream`, this test doesn't pass against the current master but passes against this branch.